cmd/cueckoo/cmd/cltrigger.go

```diff
@@ -262,24 +262,20 @@ func (c *cltrigger) triggerBuild(rev revision) error {
 		return fmt.Errorf("change %q does not know about revision %q; did you forget to run git codereview mail?", rev.changeID, commit)
 	}
 
-	return c.builder(clTriggerPayload{
-		// rev.changeID may be in the unique "project~branch~change_id" form,
-		// and we can't use that form for the workflow trigger payload
-		// as tildes are not allowed in
-		// Use the change ID alone, without any tildes.
-		ChangeID: in.ChangeID,
-
-		Ref:    revision.Ref,
-		Commit: commit,
-		Branch: in.Branch,
+	return c.builder(repositoryDispatchPayload{
+		CL:           in.ChangeNumber,
+		Patchset:     in.Revisions[in.CurrentRevision].PatchSetNumber,
```

Comment:
I assume we want `Patchset` and `Ref` to be consistent, so then this should be `revision.PatchSetNumber`.

Reply:
Wow, not sure how I missed this above. Thanks. Done